src/python/native/ext_modules.py.in

```diff
@@ -127,6 +127,8 @@ DEPENDS = [
         for source in SOURCES
 ]
 
+EXTRA_COMPILE_ARGS = ['-std=c++11']
```

Comment:
Sorry not sure what you mean, do you want the args to come through automake in general, rather than being fixed directly in the ext_modules.py ?